alf/utils/dist_utils.py

```diff
@@ -853,7 +853,9 @@ def _compute_log_prob(single_distribution, single_action):
     return total_log_probs
 
 
-def rsample_action_distribution(nested_distributions, return_log_prob=False):
+def rsample_action_distribution(nested_distributions,
+                                return_log_prob=False,
+                                disable_cache=True):
```

Comment:
I feel we should default it to False. And config it to True for ppo_cart_pole.

Reply:
> I feel we should default it to False. And config it to True for ppo_cart_pole.

Will do. 
Current test shows that even if we disable the cache, the ppo_cartpole test can still fail, which is strange. 
I'm still doing more testing on this.

Reply:
Found the root cause and pushed a new commit fixing the bug.